numpy/core/src/umath/matmul.c.src

```diff
@@ -0,0 +1,365 @@
+/* -*- c -*- */
```

Comment:
I don't think we use these markers in our other c.src files, although doing so isn't super harmful.

Reply:
`loops.c.src` has the markers

Reply:
Sounds like a convincing argument to me.